torchvision/prototype/transforms/_mixup_detection.py

```diff
@@ -0,0 +1,114 @@
+"""
+This script is used to apply the mixup transform for Object detection to the dataset.
+The code is inspired from the paper: https://arxiv.org/abs/1902.0410
+
+In a nutshell, mixup is a data augmentation technique that combines two images in the ratio of
+beta to (1-beta) and this value of beta is sampled from a beta distribution. This makes our model
+robust to the object being present in the image or not. Plus, it is kind of like a free lunch.
+"""
+from typing import Any, Dict, List, Tuple
+
+import PIL.Image
+
+import torch
+from torchvision.prototype import features
+from torchvision.prototype.transforms import Transform
+from torchvision.prototype.transforms._utils import has_any
+
+from ._augment import flatten_and_extract_data, unflatten_and_insert_data
+
+
+class MixupDetection(Transform):
+    _transformed_types = (features.is_simple_tensor, features.Image, PIL.Image)
+
+    def __init__(
+        self,
+        *,
+        alpha: float = 1.5,
+    ) -> None:
+        super().__init__()
+        self._dist = torch.distributions.Beta(torch.tensor([alpha]), torch.tensor([alpha]))
+
+    def _check_inputs(self, flat_inputs: List[Any]) -> None:
+        if has_any(flat_inputs, features.Mask):
```

Comment:
How about other unsupported types like Video?

Reply:
Resolved. Since, the current implementation doesn't support video yet. I am simply raising a TypeError if we encounter a video